go.mod

```diff
@@ -13,8 +15,9 @@ require (
 	k8s.io/api v0.18.0
 	k8s.io/apimachinery v0.18.0
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/kube-openapi v0.0.0-20190918143330-0270cf2f1c1d
-	sigs.k8s.io/controller-runtime v0.4.0
+	k8s.io/kube-openapi v0.0.0-20200204173128-addea2498afe
+	kubevirt.io/client-go v0.28.0 // indirect
```

Comment:
One last thing. This doesn't look like a dependency we need. Can you run `go mod tidy` and commit your changes?

Reply:
@sacharya Sure. Can I add the sanity phase in the makefile as well? Not sure why it was not added before. If it's ok, I can add below in Makefile
```
sanity:
	go mod tidy
	go fmt ./...
	git diff --exit-code
```

Reply:
Sure, in a separate PR.

Reply:
Done